net-print/libinklevel/files/libinklevel-0.9.3-configure.patch

```diff
@@ -0,0 +1,54 @@
+From: Boyuan Yang <byang@debian.org>
```

Comment:
links as always at the top ;)

(also may want to run iwdevtools' scrub-patch)

Reply:
oh sure, will add a link. I prefer to keep patch metadata if I'm using e.g. Debian or Fedora patch

Reply:
oh you already added the patch... and I was wondering why I can't find the open bug :smile:

Reply:
I'm sorry! I thought the package name looked familiar!

Reply:
np, most important thing is that it's fixed now ;)